modules/conversion-center/widgets/link-in-bio.php

```diff
@@ -26,6 +26,34 @@
 class Link_In_Bio extends Widget_Base {
 	const TAB_ADVANCED = 'advanced-tab-custom';
 
+	protected static $platform_icon_mapping = [
+		'Facebook'    => 'fab fa-facebook',
+		'Instagram'   => 'fab fa-instagram',
+		'LinkedIn'    => 'fab fa-linkedin-in',
+		'Pinterest'   => 'fab fa-pinterest',
+		'TikTok'      => 'fab fa-tiktok',
+		'X (Twitter)' => 'fab fa-x-twitter',
+		'YouTube'     => 'fab fa-youtube',
+		'Apple Music' => 'fab fa-apple',
+		'Behance'     => 'fab fa-behance',
+		'Dribble'     => 'fab fa-dribbble',
+		'Spotify'     => 'fab fa-spotify',
+		'SoundCloud'  => 'fab fa-soundcloud',
+		'Vimeo'       => 'fab fa-vimeo-v',
+		'Waze'        => 'fab fa-waze',
+		'WhatsApp'    => 'fab fa-whatsapp',
+		'Messenger'   => 'fab fa-facebook-messenger',
+		'Telephone'   => 'fas fa-phone-alt',
+		'Email'       => 'fas fa-envelope',
+	];
```

Comment:
@nicoladj77 Can I ask you to move this to a separate class (provider pattern)?

Example: https://github.com/elementor/elementor-pro/blob/main/modules/display-conditions/classes/comparator-provider.php

Can you replace all the usages of these Social network names in the controls with `consts` as well? It's easy to make a mistake with all these string names :)

Reply:
@louiswol94 refactored